lib/fs.js

```diff
@@ -3274,10 +3275,50 @@ defineLazyProperties(
 );
 
 ObjectDefineProperties(fs, {
```

Comment:
Can we have a test for it? We usually do something like https://github.com/nodejs/node/pull/55175/files#diff-4f2fa7a6febf9021fb93df3ec31e51b3bcbfb232a2b42bab8a1f7b496700646eR10